xonsh/color_tools.py

```diff
@@ -397,17 +397,19 @@ def color_dist(x, y):
     return math.sqrt((x[0]-y[0])**2 + (x[1]-y[1])**2 + (x[2]-y[2])**2)
 
 
-def find_closest_color(x, pallette):
-    return min(sorted(pallette.keys())[::-1],
-               key=lambda k: color_dist(x, pallette[k]))
+def find_closest_color(x, palette):
+    return min(sorted(palette.keys())[::-1],
+               key=lambda k: color_dist(x, palette[k]))
 
 
-def make_pallete(strings):
-    """Makes a color pallete from a colection of strings."""
-    pallette = {}
+def make_palette(strings):
+    """Makes a color palette from a collection of strings."""
+    palette = {}
     for s in strings:
         while '#' in s:
             _, t = s.split('#', 1)
             t, _, s = t.partition(' ')
-            pallette[t] = rgb_to_ints(t)
-    return pallette
+            palette[t] = rgb_to_ints(t)
+    return palette
+
+make_pallette = make_palette
```

Comment:
Actually, it looks like the following might be the more Pythonic way of doing this:

```python
# TODO: deprecated in 0.5.10, remove in 0.6.0.
def make_pallete(*args, **kwargs):
    """Makes a color palette from a collection of strings.

    Deprecated since version 0.5.10. Use make_palette instead.
    """
    import warnings

    make_palette(*args, **kwargs)

    warnings.warn(
        "make_pallete has been deprecated in version 0.5.10 and will be "
        "removed in 0.6.0. Please use make_palette instead.",
        DeprecationWarning)
```